code/game/machinery/dance_machine.dm

```diff
@@ -16,10 +16,10 @@
 	var/list/spotlights = list()
 	var/list/sparkles = list()
 	var/static/list/songs = list(
-		new /datum/track("Engineering's Basic Beat", 					'sound/misc/disco.ogg', 	600, 	5),
-		new /datum/track("Engineering's Domination Dance", 				'sound/misc/e1m1.ogg', 		950, 	6),
-		new /datum/track("Engineering's Superiority Shimmy", 			'sound/misc/paradox.ogg', 	2400, 	4),
-		new /datum/track("Engineering's Ultimate High-Energy Hustle",	'sound/misc/boogie2.ogg',	1770, 	5),
+		new /datum/track("Basic Beat",				 					'sound/misc/disco.ogg', 	600, 	5),
+		new /datum/track("Domination Dance",			 				'sound/misc/e1m1.ogg', 		950, 	6),
+		new /datum/track("Superiority Shimmy", 							'sound/misc/paradox.ogg', 	2400, 	4),
+		new /datum/track("Ultimate High-Energy Hustle",					'sound/misc/boogie2.ogg',	1770, 	5),
```

Comment:
These whitespace changes, reeeEE

Reply:
Also you should copy and paste everything here into the overridden songs var in modular_citadel.